index.html

```diff
@@ -334,16 +334,42 @@ <h2 id="setup">Setup</h2>
   <div class="row">
     <div class="col-md-4">
       <h4 id="shell-windows">Windows</h4>
-      <p>
-        Download the Git for Windows
-	<a href="https://git-for-windows.github.io/">installer</a>. Run the
-        installer. <strong>Important: on the 6th page of the installation wizard (the
-        page titled `Configuring the terminal emulator...`) select `Use Windows'
-        default console window`. If you forgot to do this programs that you need
-        for the workshop will not work properly. If this happens rerun the
-        installer and select the appropriate option.</strong> This will provide you
-        with both Git and Bash in the Git Bash program.
-      </p>
+      <ol>
+        <li>Download the Git for Windows <a href="https://git-for-windows.github.io/">installer</a>.</li>
+        <li>Run the installer and follow the steps bellow:
+          <ol>
+            <!-- Git 2.6.1 Setup -->
+            <!-- Welcome to the Git Setup Wizard -->
+            <li>Click on "Next".</li>
+            <!-- Information -->
+            <li>Click on "Next".</li>
+            <!-- Select Destination Location -->
+            <li>Click on "Next".</li>
+            <!-- Select Components -->
+            <li>Click on "Next".</li>
+            <!-- Select Start Menu Folder -->
+            <li>Click on "Next".</li>
+            <!-- Adjusting your PATH environment -->
+            <li>
+              <strong>
+                Select "Use Git from the Windows Command Prompt" and click on "Next".
+              </strong>
+                If you forgot to do this programs that you need for the workshop will not work properly.
+                If this happens rerun the installer and select the appropriate option.
+            </li>
+            <!-- Configuring the line ending conversions -->
+            <li>Click on "Next".</li>
+            <!-- Configuring the terminal emulator to use with Git Bash -->
+            <li>Click on "Next".</li>
```

Comment:
@ethanwhite there is no "Use Windows' default console window":

![swc-git](https://cloud.githubusercontent.com/assets/1506457/10366143/baa02e1a-6d9e-11e5-9bb6-59b70baff4fe.png)

I think they change the text at some point.